### DISCUSSION TITLE
Move the setOutput function to a separate package.

### DISCUSSION THREAD

**Discussion by Dima-Dim** [PaulHatch/semantic-version#119] [General]
Then it could be used not only as a GitHub Action, but also to get versions in other situations.

**Comment by PaulHatch** [comment]
I assume you mean publishing the action as a stand-alone package, such that it can be invoked from non-GitHub actions? This was my original plan for version 5, but I haven't gotten around to it so far.

**Comment by Dima-Dim** [comment]
The idea is to have a JS package that can run in semver, wherever needed, independent of GitHub and Actions.

**Comment by PaulHatch** [comment]
Yes, this is something I'd like to do in the future. This action was originally based on this script: https://gist.github.com/PaulHatch/6ad6d03222a8247f8a0fb9f19207d4c3

I run a number of non-GitHub CI/CD setups, most use a version of this script at the moment, but I'd like the option to use this action just like this, as a stand-alone package.

**Comment by PaulHatch** [comment]
I see should have used the "create issue from discussion" option, but anyway I added ticket #123 to track this. I am thinking to put this in v6.
